Sources/FluentKitExtras/SwiftCompatibility.swift

```diff
@@ -1,9 +1,18 @@
 import FluentKit
 
 #if compiler(>=6.2)
-@_marker public protocol _SQLKitExtrasSendableMetatype: SendableMetatype {}
+public typealias _SQLKitExtrasSendableMetatype = SendableMetatype
 #else
-@_marker public protocol _SQLKitExtrasSendableMetatype {}
+public typealias _SQLKitExtrasSendableMetatype = Any
 #endif
 
-extension FluentKit.FieldProperty: _SQLKitExtrasSendableMetatype {}
```

Comment:
Removed this one?

Reply:
Yes. Once the protocol became a `typealias`, it wasn't needed; the `Sendable` on the original property covered it all along. Now the addition is only made to the property wrappers that aren't already `Sendable` (it's still not a great solution, but Fluent 4 is a hopeless mess for Concurrency anyway).